### DISCUSSION TITLE
Lockfiles for Libraries, to commit or not?

### DISCUSSION THREAD

**Discussion by jordantshaw** [pdm-project/pdm#2585] [Q&A]
I've always been in the mindset of not committing lock files for libraries, only for applications.  What are your thoughts? Does PDM recommend committing lockfiles for libraries?

**Comment by pawamoy** [comment]
I don't think it does. I write libraries (that are also tools) and I never commit lock files. In fact, I recently started questioning whether I actually need lock files at all :sweat_smile:
